File: solutions/go/pangram/1/pangram.go
package pangram

func IsPangram(input string) bool {
    var counts [26]uint8
    for _, c := range input {
        if c >= 'A' && c <= 'Z' {
            counts[c - 'A']++
        } else if c >= 'a' && c <= 'z' {
            counts[c - 'a']++
        }
    }
    for _, c := range counts {
        if c == 0 {
            return false
        }
    }
	return true
}

File: solutions/javascript/pangram/1/pangram.js
/** @param {string} str */
export const isPangram = (str) => {
    const counts = {};
    for (const char of str.toLowerCase()) {
        if (char >= 'a' && char <= 'z') {
            counts[char] = (counts[char] || 0) + 1;
        }
    }
    return Object.keys(counts).length === 26;
};
